dev-docs/bidders/33across.md

```diff
@@ -5,13 +5,19 @@ description: Prebid 33Across Bidder Adapter
 pbjs: true
 pbs: true
 biddercode: 33across
-media_types: banner, video
+media_types: banner, video, no-native
```

Comment:
this list is supported media types only so not including native means no support for native. Can you update this for consistency across bidders 🙏

Reply:
done!